styles.css

```diff
@@ -0,0 +1,179 @@
+* {
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+}
+
+body {
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+  position: realative;
```

Comment:
- [Optional] Please, type the properties and values correctly, and remove unusable codes. 🥇